### ISSUE TITLE
Sort new cards by field

### ISSUE THREAD

**Issue by sohilsayed** [ankidroid/Anki-Android#11610]
Add the ability to sort new cards by field like in the cards browser

**Comment by mikehardy** [comment]
Hi there!
Can you be more specific about exactly the steps that get us to the point where you want the change, and how you would have the feature work?
Is this possible in AnkiDesktop?

**Comment by sohilsayed** [comment]
Sorry for not being clear i meant reposition new cards by a certain field
Like having new cards appear According to rank field

**Comment by mikehardy** [comment]
I don't believe that is supported upstream - if you can verify this is possible in Anki Desktop, then we may reopen this as an anki compatibility issue, assuming it is not possible yet upstream then the best path forward is to request the feature in https://forums.ankiweb.net to see if it may be implemented, then we follow here
